user_guides/jakartaee/src/main/jbake/content/config.adoc

```diff
@@ -113,83 +113,83 @@ JavaTest to run the sample tests.
 See link:using.html#GBFWO[Chapter 7, "Executing Tests,"] for information
 about Starting JavaTest and running tests.
 
-[[GEWTQ]][[configuring-a-jakarta-ee-9-server]]
+[[GEWTQ]][[configuring-a-jakarta-ee-10-server]]
 
-5.3 Configuring a Jakarta EE 9.1 Server
+5.3 Configuring a Jakarta EE 10 Server
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Usually there are warnings about the ~ line not matching the heading. Can trim the line to match the heading here and at other places. I dont think this can ruin the format but good to avoid these warnings.